v2/clients/http/common_test.go

```diff
@@ -0,0 +1,81 @@
+//
+// Copyright (C) 2020 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package http
+
+import (
+	"context"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/clients/http/utils"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+)
+
+const (
+	TestUnexpectedMsgFormatStr   = "unexpected result, active: '%s' but expected: '%s'"
+	TestErrorMsgFromTargetClient = "received response: {%v}. error: %s"
+)
+
+func TestGetConfig(t *testing.T) {
+	ts := newTestServer(t, v2.ApiConfigRoute, common.ConfigResponse{})
+	defer ts.Close()
+
+	gc := NewCommonClient(utils.NewClient(ts.URL))
+	response, err := gc.Configuration(context.Background())
+	if err != nil {
+		t.Errorf(TestErrorMsgFromTargetClient, response, err)
+	}
+}
```

Comment:
Need to also validate response is as expected. 
```
        expected :=  common.ConfigResponse{}
	ts := newTestServer(t, v2.ApiConfigRoute,  common.ConfigResponse{})
	defer ts.Close()

        .....

       assert.Equal(expected, response)
```
Same for other tests.

Reply:
validation of response added